evogateway.py

```diff
@@ -1210,7 +1210,7 @@ def fault_log(msg):
     elif device_type_id == 0x05:
       device_type =  "DHW"
     elif device_type_id == 0x00:
-      device_type = "CONTROLLER"
```

Comment:
This is correct as a _controller_ I think. Pls see https://github.com/domoticz/domoticz/blob/851da37f7abb99d0a6060945a48ebd49772f57cc/hardware/EvohomeRadio.cpp#L1759